ksqldb-engine/src/main/java/io/confluent/ksql/execution/pull/HARouting.java

```diff
@@ -251,25 +225,6 @@ private KsqlNode getNodeForRound(
     return location.getNodes().get(0);
   }
 
-  @SuppressWarnings("ParameterNumber")
-  @VisibleForTesting
-  interface RouteQuery {
```

Comment:
I guess this used to be there for some kind of testing purposes, but they didn't exist anymore so I just simplified the code.